src/engine/audio/SoundCodec.cpp

```diff
@@ -102,7 +102,7 @@ AudioData LoadSoundCodec(std::string filename)
 		return AudioData();
 	}
 
-	audioLogs.Warn("Sound file %s not found.", filename);
+	audioLogs.Notice("Sound file '%s' not found.", filename);
```

Comment:
I would not be surprised is the given warning is there to tell us some code is trying to load a file without checking if it exists first, so silencing the warning would not be the right fix.

Reply:
i.e. this is probably better to take care of that in another PR, I was planning to investigate on this but I don't own this, but I know there is some code trying to load optional file the required way in other places (like the model code), it may be a game issue, not an engine one.

Reply:
see 7774ff80c78f2083e23027c7a24561f07b64cf9f that is expected, the code trying to use a sound file that does not exist has to be fixed instead, so can you remove this commit? If you want I may look myself for why the code is trying to load files that does not exist, I'll was going to do it one day in any way.

Reply:
Yeah I know, I asked for it :) Didn't know it would lead to warning spam though. I've filed https://github.com/Unvanquished/Unvanquished/issues/1110 so the issue will not be buried, but in the meantime before anyone gets to looking at it, let's silence the warning.